File: pulumi-infra-project/pulumi/examples/python/digitalocean_droplet_example.py
"""
Pulumi DigitalOcean Droplet Example (Python)

Setup:
  pip install pulumi pulumi_digitalocean
Usage:
  pulumi up
"""
import pulumi
import pulumi_digitalocean as do

droplet = do.Droplet('example-droplet',
    image='ubuntu-22-04-x64',
    region='nyc3',
    size='s-1vcpu-1gb'
)
pulumi.export('droplet_id', droplet.id)
pulumi.export('droplet_ip', droplet.ipv4_address)

File: pulumi-infra-project/pulumi/examples/typescript/digitalocean_droplet_example.ts
// Pulumi DigitalOcean Droplet Example (TypeScript)
import * as pulumi from "@pulumi/pulumi";
import * as digitalocean from "@pulumi/digitalocean";

const droplet = new digitalocean.Droplet("example-droplet", {
    image: "ubuntu-22-04-x64",
    region: "nyc3",
    size: "s-1vcpu-1gb"
});

export const dropletId = droplet.id;
export const dropletIp = droplet.ipv4Address;
